java/src/main/kotlin/io/spine/validation/java/GenerationContext.kt

```diff
@@ -134,6 +162,14 @@ internal constructor(
      */
     val isElement: Boolean = elementReference != null
 
+    init {
+        System.err.println(TypeSystem::class.memberProperties.find { it.name == "files" }!!.let {
+            it.isAccessible = true
+            val files = it.get(typeSystem) as Set<ProtobufSourceFile>
+            files.map { it.file.path.value }.sorted().joinToString("\n")
```

Comment:
Why do we have a system-dependent separator here? I think we have already at least one place in ProtoData, which does such a thing. Can we have a diagnostic utility instead?